test/parallel/test-assert-deep.js

```diff
@@ -935,16 +935,18 @@ assert.deepStrictEqual(obj1, obj2);
   );
   util.inspect.defaultOptions = tmp;
 
-  const invalidTrap = new Proxy([1, 2, 3], {
-    ownKeys() { return []; }
-  });
-  assert.throws(
-    () => assert.deepStrictEqual(invalidTrap, [1, 2, 3]),
-    {
-      name: 'TypeError',
-      // message: "'ownKeys' on proxy: trap result did not include 'length'"
-    }
-  );
+  // [browserify] Safari fails this test. I'm not sure why, Chrome and Firefox pass.
+  // @BridgeAR is it ok to comment out this test?
+  // const invalidTrap = new Proxy([1, 2, 3], {
+  //   ownKeys() { return []; }
+  // });
```

Comment:
I guess the reason that Safari fails is the proxy. They might not support it properly. It should not hurt not to test this.

Reply:
Great, thanks!